README.md

```diff
@@ -39,3 +39,25 @@ git checkout main
 git pull
 release-it # Follow the instructions
 ```
+
+## Cloud nuke CLI
```

Comment:
should be in a separate readme (with ref from the main readme)

Reply:
done